app.rb

```diff
@@ -2,6 +2,8 @@
 require './modules/book_handle.rb'
```

Comment:
- [ ] It would be better if we use `require_relative` when we want to refer to files in our current directory or path. We can only use `require` when we want to use the gems installed in our gemfile :man_technologist: